### PR TITLE
update immutable rule API

### PR DESCRIPTION
**Pull request by wy65701436**
1, unify disable and enable
2, fix update rule error

Signed-off-by: wang yan <wangyan@vmware.com>

### CODE REVIEW

File: src/pkg/immutabletag/dao/immutable.go [hunk 1]
```diff
@@ -76,7 +76,7 @@ func (i *immutableRuleDao) QueryImmutableRuleByProjectID(projectID int64) ([]mod
 // QueryEnabledImmutableRuleByProjectID get all enabled immutable rule by project
 func (i *immutableRuleDao) QueryEnabledImmutableRuleByProjectID(projectID int64) ([]model.ImmutableRule, error) {
 	o := dao.GetOrmer()
-	qs := o.QueryTable(&model.ImmutableRule{}).Filter("ProjectID", projectID).Filter("Enabled", true)
+	qs := o.QueryTable(&model.ImmutableRule{}).Filter("ProjectID", projectID).Filter("Disabled", true)
```
Comment (stonezdj) [inline src/pkg/immutabletag/dao/immutable.go]: the filter should changed to disabled=false

File: src/pkg/immutabletag/dao/immutable_test.go [hunk 2]
```diff
@@ -62,7 +62,7 @@ func (t *immutableRuleDaoTestSuite) TestEnableImmutableRule() {
 	newIr, err := t.dao.GetImmutableRule(id)
 
 	t.require.Nil(err)
-	t.require.False(newIr.Enabled, "Failed to disable the immutable rule")
+	t.require.False(newIr.Disabled, "Failed to disable the immutable rule")
```
Comment (stonezdj) [inline src/pkg/immutabletag/dao/immutable_test.go]: It should be True?

File: src/pkg/immutabletag/manager_test.go [hunk 3]
```diff
@@ -110,7 +110,7 @@ func (m *managerTestingSuite) TestQueryImmutableRuleByProjectID() {
 		{
 			ID:        1,
 			ProjectID: 1,
-			Enabled:   true,
+			Disabled:  true,
```
Comment (stonezdj) [inline src/pkg/immutabletag/manager_test.go]: changed to Disabled=false

File: src/pkg/immutabletag/dao/immutable.go (hunk 1 above)
Comment (wy65701436) [reply src/pkg/immutabletag/dao/immutable.go]: ok, thanks.

File: src/pkg/immutabletag/dao/immutable_test.go (hunk 2 above)
Comment (wy65701436) [reply src/pkg/immutabletag/dao/immutable_test.go]: updated

**Pull request merged**